src/views/ContinuationIn/ContinuationInReviewConfirm.vue

```diff
@@ -73,7 +73,6 @@
 
     <!-- Continuation Effective Date and Time -->
     <section
-      v-if="(getFilingStatus === FilingStatus.DRAFT) || getEffectiveDateTime.isFutureEffective"
```

Comment:
Previously we hid some sections if this was a Draft. Now, draft applications only see "step 0". In other states, we can show these sections normally.